diff --git a/install/m4/x_ac_qe_f90.m4 b/install/m4/x_ac_qe_f90.m4
index 91d1740..32deba1 100644
--- a/install/m4/x_ac_qe_f90.m4
+++ b/install/m4/x_ac_qe_f90.m4
@@ -239,9 +239,11 @@ if test "$use_openmp" -eq 1 ; then
   try_ldflags="$try_ldflags $try_ldflags_openmp"
 fi
 
+foxflags=-fPIC
+
 if test "$fflags" = ""   ; then fflags=$try_fflags     ; fi
 if test "$f90flags" = "" ; then f90flags=$try_f90flags ; fi
-if test "try_foxflags" != ""; then foxflags=$try_foxflags; fi
+if test "try_foxflags" != ""; then foxflags="$try_foxflags $foxflags"; fi
 if test "$fflags_noopt" = ""   ; then fflags_noopt=$try_fflags_noopt     ; fi
 if test "$fflags_nomain" = ""   ; then fflags_nomain=$try_fflags_nomain     ; fi
 
